packages/vue-app/template/server.js

```diff
@@ -15,9 +15,9 @@ Vue.component('NLink', NuxtLink)
 const noopApp = () => new Vue({ render: h => h('div') })
 
 const createNext = ssrContext => (opts) => {
+  // If static target, render on client-side
   ssrContext.redirected = opts
-  // If nuxt generate
-  if (!ssrContext.res) {
+  if (ssrContext.target === 'static' || !ssrContext.res) {
```

Comment:
Can we use `TARGETS` constant here as well?

Reply:
Actually it's inside Webpack bundle, I don't know if we should import the constants there? 🤔

Reply:
It is server bundle and webpack should treeshake esm imports like this. Anyway just a minor comment :)